### ISSUE TITLE
Do not add new line on AND when it is used with BETWEEN

### ISSUE THREAD

**Issue by SscSPs** [zeroturnaround/sql-formatter#96]
https://github.com/zeroturnaround/sql-formatter/blob/2f3e9a88441a7084f56aedf32c5ec26b5362389d/src/languages/StandardSqlFormatter.js#L62

The query: 
```
select * from user where id between 10 and 20;
```
is formatted to 
```
select
  *
from
  user
where
  id between 10
  and 20;
```

which is wrong as it should be converted to:
```
select
  *
from
  user
where
  id between 10 and 20;
```

**Comment by nene** [comment]
A valid problem. Feel free to send a pull request to fix it.

**Comment by nene** [comment]
This is now fixed (at least for the basic case) in `4.0.0-beta.2` release.
